src/java/org/apache/cassandra/tools/nodetool/clustermetadata/DescribeCMS.java

```diff
@@ -16,29 +16,29 @@
  * limitations under the License.
  */
 
-package org.apache.cassandra.tools.nodetool;
+package org.apache.cassandra.tools.nodetool.clustermetadata;
 
 import java.util.Map;
 
 import io.airlift.airline.Command;
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool;
 
-@Command(name = "describecms", description = "Describe the current Cluster Metadata Service")
+@Command(name = "describe", description = "Describe the current Cluster Metadata Service")
 public class DescribeCMS extends NodeTool.NodeToolCmd
 {
     @Override
     protected void execute(NodeProbe probe)
     {
         Map<String, String> info = probe.getCMSOperationsProxy().describeCMS();
-        System.out.printf("Cluster Metadata Service:%n");
-        System.out.printf("Members: %s%n", info.get("MEMBERS"));
-        System.out.printf("Is Member: %s%n", info.get("IS_MEMBER"));
-        System.out.printf("Service State: %s%n", info.get("SERVICE_STATE"));
-        System.out.printf("Is Migrating: %s%n", info.get("IS_MIGRATING"));
-        System.out.printf("Epoch: %s%n", info.get("EPOCH"));
-        System.out.printf("Local Pending Count: %s%n", info.get("LOCAL_PENDING"));
-        System.out.printf("Commits Paused: %s%n", info.get("COMMITS_PAUSED"));
-        System.out.printf("Replication factor: %s%n", info.get("REPLICATION_FACTOR"));
+        output.out.printf("Cluster Metadata Service:%n");
```

Comment:
this is a good change, I think this enable us in the future to test the output in unit tests as it is not writing it to System.out but to probe's output